orchagent/port_rates.lua

```diff
@@ -83,6 +193,21 @@ local function compute_rate(port)
             redis.call('HSET', rates_table_name .. ':' .. port, 'TX_PPS', tx_pps_new)
             redis.call('HSET', state_table, 'INIT_DONE', 'DONE')
         end
+
+        -- only do the calculation when all info present
+        
+        if fec_corr_bits and fec_uncorr_frames and lanes_count and serdes_speed then
+            local fec_corr_bits_last = redis.call('HGET', rates_table_name .. ':' .. port, 'SAI_PORT_STAT_IF_FEC_CORRECTED_BITS_last')
+            local fec_uncorr_frames_last = redis.call('HGET', rates_table_name .. ':' .. port, 'SAI_PORT_STAT_IF_FEC_NOT_CORRECTABLE_FARMES_last')
+
+            local serdes_rate_total = lanes_count * serdes_speed * delta / 1000
```

Comment:
@vincentpcng this is actually total bits during the delta period

Reply:
@vincentpcng i was referring to the variable name `serdes_rate_total` which actually could be renamed `total_bits` since its not `rate`